lib/modules/dosomething/dosomething_signup/includes/SignupsController.php

```diff
@@ -2,6 +2,8 @@
 
 class SignupsController extends EntityAPIController {
 
+  protected $ignore_global_user = FALSE;
```

Comment:
Yeah, I was just afraid to change the current `save()` method too much and potentially break signups which are kind of a crucial feature. I make notes to eventually find a better solution for this, since it's not ideal, but figured not worth modifying if the `/magicks` experiment doesn't work out.